exercises/concepts/jeffrey-juicy-lasagna/.meta/design.md

```diff
@@ -0,0 +1,53 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student some of the basics of programming with Powershell.
+
+## Learning objectives
+
+- Know what a variable is and how to define it in many ways.
+- Know how to update or reassign variable.
+- Know that variable can have type conversion.
+- Know what a `function` is, its basic interface and how to define one via `function`.
+- Know how to add one or more `parameters` to a function definition.
+- Know that `parameters` definitions can have type annotations.
```

Comment:
I didn't see either of these in the exercise instructions, but maybe in the links?

Reply:
Oh yeah, that 2nd part should be moved to the advanced function concept.